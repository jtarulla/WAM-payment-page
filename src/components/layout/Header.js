import React, { Component } from 'react';
import logo from '../../assets/logo.jpg';
import help from '../../assets/navbar-icons/help.png';
import signin from '../../assets/navbar-icons/signin.png';
import arrow from '../../assets/navbar-icons/arrow.png';

export class Header extends Component {
	openMenu = () => {
		var element = document.getElementById('navbar-list');
		if (element.className === 'navbar-list') {
			element.className += ' responsive';
		} else {
			element.className = 'navbar-list';
		}
	};

	render() {
		return (
			<header className='header'>
				<div className='logo'>
					<a
						href='https://www.wearemarketing.com/es/'
						target='_blank'
						rel='noopener noreferrer'
					>
						<img src={logo} alt='WAM' />
					</a>
				</div>

				<div className='navbar'>
					<div className='navbar-list' id='navbar-list'>
						<ul>
							<li className='navbar-item'>
								<a className='item-name-help' href='#help'>
									<img src={help} alt='help' />
									<span>ayuda</span>
								</a>
							</li>

							<li className='navbar-item'>
								<a className='item-name-signin' href='#signin'>
									<img
										className='item-name-signin-img'
										src={signin}
										alt='signin'
									/>
									<span>sign in</span>
								</a>
							</li>

							<li className='navbar-item'>
								<a className='item-name-es' href='#es'>
									es
									<img src={arrow} alt='arrow' />
								</a>
							</li>
						</ul>
						<div className='menu' onClick={this.openMenu}>
							Men√∫
						</div>
						<div className='menu-close' onClick={this.openMenu}>
							&#10005;
						</div>
					</div>
				</div>
			</header>
		);
	}
}

export default Header;
