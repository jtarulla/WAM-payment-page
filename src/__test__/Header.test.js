import { shallow } from 'enzyme';
import React from 'react';
import Header from '../components/layout/Header';

describe('Header component', () => {
	it('renders Header component', () => {
		expect(shallow(<Header />)).toMatchSnapshot();
	});
});
