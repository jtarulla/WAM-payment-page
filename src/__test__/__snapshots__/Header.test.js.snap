// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders Header component 1`] = `ShallowWrapper {}`;
